google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITDatabaseAdminTest.java

```diff
@@ -260,9 +259,8 @@ public void updateDatabaseInvalidFieldsToUpdate() {
     OperationFuture<Database, UpdateDatabaseMetadata> op =
         dbAdminClient.updateDatabase(databaseToUpdate);
 
-    ExecutionException e =
-        assertThrows(ExecutionException.class, () -> op.get(5, TimeUnit.MINUTES));
-    assertThat(e.getMessage()).contains("Invalid field mask");
+    SpannerException e = assertThrows(SpannerException.class, () -> op.get(5, TimeUnit.MINUTES));
```

Comment:
I'm pretty sure that the error that is returned by `op.get()` will be an `ExecutionException`, and not a `SpannerException`. However, the underlying cause of the `ExecutionException` should be a `SpannerException`.

(The build also fails because this assertion fails, as the returned exception is an `ExecutionException`).

Reply:
Thanks, I have fixed the asserts.